FAIL: TestAccessiblePrompter/AuthToken_-_blank_input_returns_error

### Describe the bug
Hello,
we're building `gh` for openSUSE a SUSE Linux and new error in tests just appeared:

```
[   91s] ?   	github.com/cli/cli/v2/internal/keyring	[no test files]
[   93s] --- FAIL: TestAccessiblePrompter (1.04s)
[   93s]     --- FAIL: TestAccessiblePrompter/AuthToken_-_blank_input_returns_error (1.00s)
[   93s]         expect.go:76: Failed to find [" \r\n\r\n"] in "\r\nPaste your authentication token: 12345abcdefg\r\n\r\n\r\n": read |0: i/o timeout
[   93s] FAIL
[   93s] FAIL	github.com/cli/cli/v2/internal/prompter	1.049s
```

### Affected version

`v2.72.0`

### Steps to reproduce the behavior

```
cd /home/abuild/rpmbuild/BUILD
cd cli-2.72.0
GOFLAGS='-buildmode=pie -trimpath -mod=vendor -modcacherw'
make test
```

### Expected vs actual behavior

The previous version (v2.70.0 in our case) passed the tests just fine.
I'm aware that this is new functionality and that the error might be specific to Open Build Service which we use for packaging.

### Logs

The full log is available on [build.opensuse.org/package/live_build_log/home:pdostal:branches:devel:tools:scm/gh/openSUSE_Tumbleweed/x86_64](https://build.opensuse.org/package/live_build_log/home:pdostal:branches:devel:tools:scm/gh/openSUSE_Tumbleweed/x86_64)

--- a/internal/prompter/accessible_prompter_test.go
+++ b/internal/prompter/accessible_prompter_test.go
@@ -32,6 +32,9 @@ import (
 // are sufficient to ensure that the accessible prompter behaves roughly as expected
 // but doesn't mandate that prompts always look exactly the same.
 func TestAccessiblePrompter(t *testing.T) {
+
+	beforePasswordSendTimeout := 20 * time.Microsecond
+
 	t.Run("Select", func(t *testing.T) {
 		console := newTestVirtualTerminal(t)
 		p := newTestAccessiblePrompter(t, console)
@@ -147,6 +150,9 @@ func TestAccessiblePrompter(t *testing.T) {
 			_, err := console.ExpectString("Enter password")
 			require.NoError(t, err)
 
+			// Wait to ensure huh has time to set the echo mode
+			time.Sleep(beforePasswordSendTimeout)
+
 			// Enter a number
 			_, err = console.SendLine(dummyPassword)
 			require.NoError(t, err)
@@ -210,6 +216,9 @@ func TestAccessiblePrompter(t *testing.T) {
 			_, err := console.ExpectString("Paste your authentication token:")
 			require.NoError(t, err)
 
+			// Wait to ensure huh has time to set the echo mode
+			time.Sleep(beforePasswordSendTimeout)
+
 			// Enter some dummy auth token
 			_, err = console.SendLine(dummyAuthToken)
 			require.NoError(t, err)
@@ -242,6 +251,9 @@ func TestAccessiblePrompter(t *testing.T) {
 			// Expect an error message
 			_, err = console.ExpectString("token is required")
 			require.NoError(t, err)
+
+			// Wait to ensure huh has time to set the echo mode
+			time.Sleep(beforePasswordSendTimeout)
 
 			// Now enter some dummy auth token to return control back to the test
 			_, err = console.SendLine(dummyAuthTokenForAfterFailure)

